{
    "data": [
        {
            "test_name": "python.simplescanviewer.RectanglePlotTest.RectanglePlotTest",
            "latest_failed_build": "1778",
            "agent_name": "Linux",
            "last_failed_outcome": "Failed",
            "pipeline_name": "pull_requests-conda-linux",
            "problem_count": 2,
            "past_outcome": {
                "build_keys": [
                    "1781",
                    "1780",
                    "1779",
                    "1778",
                    "1777",
                    "1776",
                    "1775",
                    "1774",
                    "1773",
                    "1772",
                    "1771",
                    "1770",
                    "1769",
                    "1768",
                    "1767",
                    "1766",
                    "1765",
                    "1764",
                    "1763",
                    "1762",
                    "1761",
                    "1760",
                    "1759",
                    "1758",
                    "1757",
                    "1756",
                    "1755",
                    "1754",
                    "1753",
                    "1752"
                ],
                "past_outcome_list": [
                    "None",
                    "Passed",
                    "Passed",
                    "Failed",
                    "Failed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "None",
                    "Passed",
                    "None",
                    "Passed",
                    "None",
                    "None",
                    "None",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "None",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "None"
                ]
            },
            "last_stack_traces": {
                "test_name": "python.simplescanviewer.RectanglePlotTest.RectanglePlotTest",
                "build": "1778",
                "agent_name": "Linux",
                "trial": 3,
                "outcome": "Failed",
                "test_time": "9.06",
                "stacktrace": "FrameworkManager-[Notice] Welcome to Mantid 6.5.20221102.1142\nFrameworkManager-[Notice] Please cite: http://dx.doi.org/10.1016/j.nima.2014.07.029 and this release: http://dx.doi.org/10.5286/Software/Mantid\nQStandardPaths: XDG_RUNTIME_DIR not set, defaulting to &#x27;/tmp/runtime-abc&#x27;\n/jenkins_workdir/workspace/pull_requests-conda-linux/build/bin/site.py:26: DeprecationWarning: the imp module is deprecated in favour of importlib; see the module&#x27;s documentation for alternative uses\n  import imp  # Avoid import loop in Python 3\n/jenkins_workdir/workspace/pull_requests-conda-linux/mambaforge/envs/mantid-developer/lib/python3.8/site-packages/h5py/__init__.py:46: DeprecationWarning: `np.typeDict` is a deprecated alias for `np.sctypeDict`.\n  from ._conv import register_converters as _register_converters\nThe global colormaps dictionary is no longer considered public API.\n..CreateEmptyTableWorkspace-[Notice] CreateEmptyTableWorkspace started\nCreateEmptyTableWorkspace-[Notice] CreateEmptyTableWorkspace successful, Duration 0.02 seconds\n.CreateWorkspace-[Notice] CreateWorkspace started\nCreateWorkspace-[Notice] CreateWorkspace successful, Duration 0.07 seconds\nCreateWorkspace-[Notice] CreateWorkspace started\nCreateWorkspace-[Notice] CreateWorkspace successful, Duration 0.04 seconds\nCreateWorkspace-[Notice] CreateWorkspace started\nCreateWorkspace-[Notice] CreateWorkspace successful, Duration 0.05 seconds\nCreateWorkspace-[Notice] CreateWorkspace started\nCreateWorkspace-[Notice] CreateWorkspace successful, Duration 0.04 seconds\nCreateWorkspace-[Notice] CreateWorkspace started\nCreateWorkspace-[Notice] CreateWorkspace successful, Duration 0.05 seconds\nCreateWorkspace-[Notice] CreateWorkspace started\nCreateWorkspace-[Notice] CreateWorkspace successful, Duration 0.03 seconds\nCreateWorkspace-[Notice] CreateWorkspace started\nCreateWorkspace-[Notice] CreateWorkspace successful, Duration 0.04 seconds\nCreateWorkspace-[Notice] CreateWorkspace started\nCreateWorkspace-[Notice] CreateWorkspace successful, Duration 0.04 seconds\n...Traceback (most recent call last):\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/mantidqtinterfaces/simplescanviewer/rectangle_plot.py&quot;, line 261, in _on_controller_updated\n    self._update_plot_values()\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/mantidqtinterfaces/simplescanviewer/rectangle_plot.py&quot;, line 274, in _update_plot_values\n    (x_line_x_axis, x_line_y_values), (y_line_x_axis, y_line_y_values) = self._compute_plot_axes()\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/mantidqtinterfaces/simplescanviewer/rectangle_plot.py&quot;, line 288, in _compute_plot_axes\n    xmin, xmax, ymin, ymax = self.plotter.image.get_extent()\nValueError: not enough values to unpack (expected 4, got 0)\nSubprocess aborted\n\n          Start  606: CrystalTest_CompositeClusterTest\n"
            }
        },
        {
            "test_name": "python.MuonQt5.tf_asymmetry_fitting_options_view_test.tf_asymmetry_fitting_options_view_test",
            "latest_failed_build": "1776",
            "agent_name": "Linux",
            "last_failed_outcome": "Flake",
            "pipeline_name": "pull_requests-conda-linux",
            "problem_count": 1,
            "past_outcome": {
                "build_keys": [
                    "1781",
                    "1780",
                    "1779",
                    "1778",
                    "1777",
                    "1776",
                    "1775",
                    "1774",
                    "1773",
                    "1772",
                    "1771",
                    "1770",
                    "1769",
                    "1768",
                    "1767",
                    "1766",
                    "1765",
                    "1764",
                    "1763",
                    "1762",
                    "1761",
                    "1760",
                    "1759",
                    "1758",
                    "1757",
                    "1756",
                    "1755",
                    "1754",
                    "1753",
                    "1752"
                ],
                "past_outcome_list": [
                    "None",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Flake",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "None",
                    "Passed",
                    "None",
                    "Passed",
                    "None",
                    "None",
                    "None",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "None",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "None"
                ]
            },
            "last_stack_traces": {
                "test_name": "python.MuonQt5.tf_asymmetry_fitting_options_view_test.tf_asymmetry_fitting_options_view_test",
                "build": "1776",
                "agent_name": "Linux",
                "trial": 1,
                "outcome": "Failed",
                "test_time": "1.32",
                "stacktrace": "QStandardPaths: XDG_RUNTIME_DIR not set, defaulting to &#x27;/tmp/runtime-abc&#x27;\n/jenkins_workdir/workspace/pull_requests-conda-linux/build/bin/site.py:26: DeprecationWarning: the imp module is deprecated in favour of importlib; see the module&#x27;s documentation for alternative uses\n  import imp  # Avoid import loop in Python 3\n.EEEEEEE\n======================================================================\nERROR: test_that_hide_normalisation_options_will_hide_all_of_the_tf_asymmetry_options (tf_asymmetry_fitting_options_view_test.TFAsymmetryFittingOptionsViewTest)\n----------------------------------------------------------------------\nAttributeError: __getattribute__\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/test/Muon/fitting_widgets/tf_asymmetry_fitting/tf_asymmetry_fitting_options_view_test.py&quot;, line 20, in setUp\n    self.view = TFAsymmetryFittingOptionsView()\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/fitting_widgets/tf_asymmetry_fitting/tf_asymmetry_fitting_options_view.py&quot;, line 23, in __init__\n    self.setupUi(self)\n  File &quot;&lt;string&gt;&quot;, line 44, in setupUi\nSystemError: &lt;built-in function connectSlotsByName&gt; returned a result with an error set\n\n======================================================================\nERROR: test_that_is_normalisation_fixed_can_be_set_to_true (tf_asymmetry_fitting_options_view_test.TFAsymmetryFittingOptionsViewTest)\n----------------------------------------------------------------------\nAttributeError: __getattribute__\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/test/Muon/fitting_widgets/tf_asymmetry_fitting/tf_asymmetry_fitting_options_view_test.py&quot;, line 20, in setUp\n    self.view = TFAsymmetryFittingOptionsView()\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/fitting_widgets/tf_asymmetry_fitting/tf_asymmetry_fitting_options_view.py&quot;, line 23, in __init__\n    self.setupUi(self)\n  File &quot;&lt;string&gt;&quot;, line 44, in setupUi\nSystemError: &lt;built-in function connectSlotsByName&gt; returned a result with an error set\n\n======================================================================\nERROR: test_that_set_tf_asymmetry_mode_will_hide_the_tf_asymmetry_options_when_set_to_false (tf_asymmetry_fitting_options_view_test.TFAsymmetryFittingOptionsViewTest)\n----------------------------------------------------------------------\nAttributeError: __getattribute__\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/test/Muon/fitting_widgets/tf_asymmetry_fitting/tf_asymmetry_fitting_options_view_test.py&quot;, line 20, in setUp\n    self.view = TFAsymmetryFittingOptionsView()\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/fitting_widgets/tf_asymmetry_fitting/tf_asymmetry_fitting_options_view.py&quot;, line 23, in __init__\n    self.setupUi(self)\n  File &quot;&lt;string&gt;&quot;, line 44, in setupUi\nSystemError: &lt;built-in function connectSlotsByName&gt; returned a result with an error set\n\n======================================================================\nERROR: test_that_set_tf_asymmetry_mode_will_show_the_tf_asymmetry_options_when_set_to_true (tf_asymmetry_fitting_options_view_test.TFAsymmetryFittingOptionsViewTest)\n----------------------------------------------------------------------\nAttributeError: __getattribute__\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/test/Muon/fitting_widgets/tf_asymmetry_fitting/tf_asymmetry_fitting_options_view_test.py&quot;, line 20, in setUp\n    self.view = TFAsymmetryFittingOptionsView()\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/fitting_widgets/tf_asymmetry_fitting/tf_asymmetry_fitting_options_view.py&quot;, line 23, in __init__\n    self.setupUi(self)\n  File &quot;&lt;string&gt;&quot;, line 44, in setupUi\nSystemError: &lt;built-in function connectSlotsByName&gt; returned a result with an error set\n\n======================================================================\nERROR: test_that_show_normalisation_options_will_show_all_of_the_tf_asymmetry_options (tf_asymmetry_fitting_options_view_test.TFAsymmetryFittingOptionsViewTest)\n----------------------------------------------------------------------\nAttributeError: __getattribute__\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/test/Muon/fitting_widgets/tf_asymmetry_fitting/tf_asymmetry_fitting_options_view_test.py&quot;, line 20, in setUp\n    self.view = TFAsymmetryFittingOptionsView()\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/fitting_widgets/tf_asymmetry_fitting/tf_asymmetry_fitting_options_view.py&quot;, line 23, in __init__\n    self.setupUi(self)\n  File &quot;&lt;string&gt;&quot;, line 44, in setupUi\nSystemError: &lt;built-in function connectSlotsByName&gt; returned a result with an error set\n\n======================================================================\nERROR: test_that_the_normalisation_can_be_set_as_expected (tf_asymmetry_fitting_options_view_test.TFAsymmetryFittingOptionsViewTest)\n----------------------------------------------------------------------\nAttributeError: __getattribute__\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/test/Muon/fitting_widgets/tf_asymmetry_fitting/tf_asymmetry_fitting_options_view_test.py&quot;, line 20, in setUp\n    self.view = TFAsymmetryFittingOptionsView()\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/fitting_widgets/tf_asymmetry_fitting/tf_asymmetry_fitting_options_view.py&quot;, line 23, in __init__\n    self.setupUi(self)\n  File &quot;&lt;string&gt;&quot;, line 44, in setupUi\nSystemError: &lt;built-in function connectSlotsByName&gt; returned a result with an error set\n\n======================================================================\nERROR: test_that_the_view_has_been_initialized_with_a_normalisation_line_edit_which_has_a_validator (tf_asymmetry_fitting_options_view_test.TFAsymmetryFittingOptionsViewTest)\n----------------------------------------------------------------------\nAttributeError: __getattribute__\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/test/Muon/fitting_widgets/tf_asymmetry_fitting/tf_asymmetry_fitting_options_view_test.py&quot;, line 20, in setUp\n    self.view = TFAsymmetryFittingOptionsView()\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/fitting_widgets/tf_asymmetry_fitting/tf_asymmetry_fitting_options_view.py&quot;, line 23, in __init__\n    self.setupUi(self)\n  File &quot;&lt;string&gt;&quot;, line 44, in setupUi\nSystemError: &lt;built-in function connectSlotsByName&gt; returned a result with an error set\n\n----------------------------------------------------------------------\nRan 8 tests in 0.222s\n\nFAILED (errors=7)\n\n          Start 2191: python.MuonQt5.tf_asymmetry_fitting_options_view_test.tf_asymmetry_fitting_options_view_test\n"
            }
        },
        {
            "test_name": "mantidqt_qt5.test_sample_shape.test_sample_shape",
            "latest_failed_build": "1776",
            "agent_name": "Linux",
            "last_failed_outcome": "Failed",
            "pipeline_name": "pull_requests-conda-linux",
            "problem_count": 1,
            "past_outcome": {
                "build_keys": [
                    "1781",
                    "1780",
                    "1779",
                    "1778",
                    "1777",
                    "1776",
                    "1775",
                    "1774",
                    "1773",
                    "1772",
                    "1771",
                    "1770",
                    "1769",
                    "1768",
                    "1767",
                    "1766",
                    "1765",
                    "1764",
                    "1763",
                    "1762",
                    "1761",
                    "1760",
                    "1759",
                    "1758",
                    "1757",
                    "1756",
                    "1755",
                    "1754",
                    "1753",
                    "1752"
                ],
                "past_outcome_list": [
                    "None",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Failed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "None",
                    "Passed",
                    "None",
                    "Passed",
                    "None",
                    "None",
                    "None",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "None",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "None"
                ]
            },
            "last_stack_traces": {
                "test_name": "mantidqt_qt5.test_sample_shape.test_sample_shape",
                "build": "1776",
                "agent_name": "Linux",
                "trial": 3,
                "outcome": "Failed",
                "test_time": "25.99",
                "stacktrace": "FrameworkManager-[Notice] Welcome to Mantid 6.5.20221102.1009\nFrameworkManager-[Notice] Please cite: http://dx.doi.org/10.1016/j.nima.2014.07.029 and this release: http://dx.doi.org/10.5286/Software/Mantid\nCreateWorkspace-[Notice] CreateWorkspace started\nCreateWorkspace-[Notice] CreateWorkspace successful, Duration 0.05 seconds\nDeleteWorkspace-[Notice] DeleteWorkspace started\nDeleteWorkspace-[Notice] DeleteWorkspace successful, Duration 0.00 seconds\n/jenkins_workdir/workspace/pull_requests-conda-linux/build/bin/site.py:26: DeprecationWarning: the imp module is deprecated in favour of importlib; see the module&#x27;s documentation for alternative uses\n  import imp  # Avoid import loop in Python 3\n/jenkins_workdir/workspace/pull_requests-conda-linux/mambaforge/envs/mantid-developer/lib/python3.8/site-packages/h5py/__init__.py:46: DeprecationWarning: `np.typeDict` is a deprecated alias for `np.sctypeDict`.\n  from ._conv import register_converters as _register_converters\n.CreateWorkspace-[Notice] CreateWorkspace started\nCreateWorkspace-[Notice] CreateWorkspace successful, Duration 0.00 seconds\nSetSample-[Notice] SetSample started\nSetSample-[Notice] SetSample successful, Duration 1.07 seconds\nDeleteWorkspace-[Notice] DeleteWorkspace started\nDeleteWorkspace-[Notice] DeleteWorkspace successful, Duration 0.00 seconds\n.CreateSampleWorkspace-[Notice] CreateSampleWorkspace started\nCreateSampleWorkspace-[Notice] CreateSampleWorkspace successful, Duration 0.00 seconds\nDeleteWorkspace-[Notice] DeleteWorkspace started\nDeleteWorkspace-[Notice] DeleteWorkspace successful, Duration 0.00 seconds\n.CreateWorkspace-[Notice] CreateWorkspace started\nCreateWorkspace-[Notice] CreateWorkspace successful, Duration 0.00 seconds\nLoadInstrument-[Notice] LoadInstrument started\nLoadInstrument-[Notice] LoadInstrument successful, Duration 0.01 seconds\nSetSample-[Notice] SetSample started\nSetSample-[Notice] SetSample successful, Duration 0.12 seconds\nDeleteWorkspace-[Notice] DeleteWorkspace started\nDeleteWorkspace-[Notice] DeleteWorkspace successful, Duration 0.00 seconds\nMatplotlib is currently using agg, which is a non-GUI backend, so cannot show the figure.\n.CreateWorkspace-[Notice] CreateWorkspace started\nCreateWorkspace-[Notice] CreateWorkspace successful, Duration 0.00 seconds\nLoadInstrument-[Notice] LoadInstrument started\nLoadInstrument-[Notice] LoadInstrument successful, Duration 0.00 seconds\nSetSample-[Notice] SetSample started\nSetSample-[Notice] SetSample successful, Duration 0.00 seconds\nDeleteWorkspace-[Notice] DeleteWorkspace started\nDeleteWorkspace-[Notice] DeleteWorkspace successful, Duration 0.00 seconds\nMatplotlib is currently using agg, which is a non-GUI backend, so cannot show the figure.\n.CreateWorkspace-[Notice] CreateWorkspace started\nCreateWorkspace-[Notice] CreateWorkspace successful, Duration 0.01 seconds\nLoadInstrument-[Notice] LoadInstrument started\nLoadInstrument-[Notice] LoadInstrument successful, Duration 0.00 seconds\nSetSample-[Notice] SetSample started\nSetSample-[Notice] SetSample successful, Duration 0.00 seconds\nDeleteWorkspace-[Notice] DeleteWorkspace started\nDeleteWorkspace-[Notice] DeleteWorkspace successful, Duration 0.00 seconds\nMatplotlib is currently using agg, which is a non-GUI backend, so cannot show the figure.\n.CreateWorkspace-[Notice] CreateWorkspace started\nCreateWorkspace-[Notice] CreateWorkspace successful, Duration 0.00 seconds\nLoadInstrument-[Notice] LoadInstrument started\nLoadInstrument-[Notice] LoadInstrument successful, Duration 0.00 seconds\nSetSample-[Notice] SetSample started\nSetSample-[Notice] SetSample successful, Duration 0.00 seconds\nDeleteWorkspace-[Notice] DeleteWorkspace started\nDeleteWorkspace-[Notice] DeleteWorkspace successful, Duration 0.00 seconds\n.CreateWorkspace-[Notice] CreateWorkspace started\nCreateWorkspace-[Notice] CreateWorkspace successful, Duration 0.00 seconds\nLoadInstrument-[Notice] LoadInstrument started\nLoadInstrument-[Notice] LoadInstrument successful, Duration 0.00 seconds\nSetSample-[Notice] SetSample started\nSetSample-[Notice] SetSample successful, Duration 0.00 seconds\nDeleteWorkspace-[Notice] DeleteWorkspace started\nDeleteWorkspace-[Notice] DeleteWorkspace successful, Duration 0.00 seconds\n.CreateWorkspace-[Notice] CreateWorkspace started\nCreateWorkspace-[Notice] CreateWorkspace successful, Duration 0.00 seconds\nLoadInstrument-[Notice] LoadInstrument started\nLoadInstrument-[Notice] LoadInstrument successful, Duration 0.00 seconds\nSetSample-[Notice] SetSample started\nSetSample-[Notice] SetSample successful, Duration 0.00 seconds\nDeleteWorkspace-[Notice] DeleteWorkspace started\nDeleteWorkspace-[Notice] DeleteWorkspace successful, Duration 0.00 seconds\n.CreateSampleWorkspace-[Notice] CreateSampleWorkspace started\nCreateSampleWorkspace-[Notice] CreateSampleWorkspace successful, Duration 0.00 seconds\nLoadSampleShape-[Notice] LoadSampleShape started\nLoadSampleShape-[Notice] LoadSampleShape successful, Duration 0.01 seconds\nDeleteWorkspace-[Notice] DeleteWorkspace started\nDeleteWorkspace-[Notice] DeleteWorkspace successful, Duration 0.00 seconds\n.CreateWorkspace-[Notice] CreateWorkspace started\nCreateWorkspace-[Notice] CreateWorkspace successful, Duration 0.00 seconds\nLoadInstrument-[Notice] LoadInstrument started\nLoadInstrument-[Notice] LoadInstrument successful, Duration 0.00 seconds\nSetSample-[Notice] SetSample started\nSetSample-[Notice] SetSample successful, Duration 0.00 seconds\nDeleteWorkspace-[Notice] DeleteWorkspace started\nDeleteWorkspace-[Notice] DeleteWorkspace successful, Duration 0.00 seconds\n.CreateWorkspace-[Notice] CreateWorkspace started\nCreateWorkspace-[Notice] CreateWorkspace successful, Duration 0.00 seconds\nLoadInstrument-[Notice] LoadInstrument started\nLoadInstrument-[Notice] LoadInstrument successful, Duration 0.00 seconds\nSetSample-[Notice] SetSample started\nSetSample-[Notice] SetSample successful, Duration 0.00 seconds\nDeleteWorkspace-[Notice] DeleteWorkspace started\nDeleteWorkspace-[Notice] DeleteWorkspace successful, Duration 0.00 seconds\n.CreateWorkspace-[Notice] CreateWorkspace started\nCreateWorkspace-[Notice] CreateWorkspace successful, Duration 0.00 seconds\nLoadInstrument-[Notice] LoadInstrument started\nLoadInstrument-[Notice] LoadInstrument successful, Duration 0.00 seconds\nSetSample-[Notice] SetSample started\nSetSample-[Notice] SetSample successful, Duration 0.00 seconds\nDeleteWorkspace-[Notice] DeleteWorkspace started\nDeleteWorkspace-[Notice] DeleteWorkspace successful, Duration 0.00 seconds\n.CreateWorkspace-[Notice] CreateWorkspace started\nCreateWorkspace-[Notice] CreateWorkspace successful, Duration 0.00 seconds\nSetSample-[Notice] SetSample started\nSetSample-[Notice] SetSample successful, Duration 0.00 seconds\nDeleteWorkspace-[Notice] DeleteWorkspace started\nDeleteWorkspace-[Notice] DeleteWorkspace successful, Duration 0.00 seconds\n.CreateSampleWorkspace-[Notice] CreateSampleWorkspace started\nCreateSampleWorkspace-[Notice] CreateSampleWorkspace successful, Duration 0.00 seconds\nLoadSampleShape-[Notice] LoadSampleShape started\nLoadSampleShape-[Notice] LoadSampleShape successful, Duration 0.01 seconds\nDeleteWorkspace-[Notice] DeleteWorkspace started\nDeleteWorkspace-[Notice] DeleteWorkspace successful, Duration 0.01 seconds\n.CreateWorkspace-[Notice] CreateWorkspace started\nCreateWorkspace-[Notice] CreateWorkspace successful, Duration 0.00 seconds\nLoadInstrument-[Notice] LoadInstrument started\nLoadInstrument-[Notice] LoadInstrument successful, Duration 0.00 seconds\nSetSample-[Notice] SetSample started\nSetSample-[Notice] SetSample successful, Duration 0.00 seconds\nDeleteWorkspace-[Notice] DeleteWorkspace started\nDeleteWorkspace-[Notice] DeleteWorkspace successful, Duration 0.00 seconds\n.CreateSampleWorkspace-[Notice] CreateSampleWorkspace started\nCreateSampleWorkspace-[Notice] CreateSampleWorkspace successful, Duration 0.02 seconds\nLoadSampleShape-[Notice] LoadSampleShape started\nLoadSampleShape-[Notice] LoadSampleShape successful, Duration 0.01 seconds\nDeleteWorkspace-[Notice] DeleteWorkspace started\nDeleteWorkspace-[Notice] DeleteWorkspace successful, Duration 0.00 seconds\n.CreateWorkspace-[Notice] CreateWorkspace started\nCreateWorkspace-[Notice] CreateWorkspace successful, Duration 0.00 seconds\nLoadInstrument-[Notice] LoadInstrument started\nLoadInstrument-[Notice] LoadInstrument successful, Duration 0.00 seconds\nSetSample-[Notice] SetSample started\nSetSample-[Notice] SetSample successful, Duration 0.00 seconds\nDeleteWorkspace-[Notice] DeleteWorkspace started\nDeleteWorkspace-[Notice] DeleteWorkspace successful, Duration 0.00 seconds\n..CreateSampleWorkspace-[Notice] CreateSampleWorkspace started\nCreateSampleWorkspace-[Notice] CreateSampleWorkspace successful, Duration 0.00 seconds\nSetUB-[Notice] SetUB started\nSetUB-[Notice] SetUB successful, Duration 0.00 seconds\n.CreateWorkspace-[Notice] CreateWorkspace started\nCreateWorkspace-[Notice] CreateWorkspace successful, Duration 0.01 seconds\nLoadInstrument-[Notice] LoadInstrument started\nLoadInstrument-[Notice] LoadInstrument successful, Duration 0.00 seconds\nSetSample-[Notice] SetSample started\nSetSample-[Notice] SetSample successful, Duration 0.00 seconds\nDeleteWorkspace-[Notice] DeleteWorkspace started\nDeleteWorkspace-[Notice] DeleteWorkspace successful, Duration 0.00 seconds\nMatplotlib is currently using agg, which is a non-GUI backend, so cannot show the figure.\n.CreateSampleWorkspace-[Notice] CreateSampleWorkspace started\nCreateSampleWorkspace-[Notice] CreateSampleWorkspace successful, Duration 0.00 seconds\nLoadSampleShape-[Notice] LoadSampleShape started\nLoadSampleShape-[Notice] LoadSampleShape successful, Duration 0.01 seconds\nDeleteWorkspace-[Notice] DeleteWorkspace started\nDeleteWorkspace-[Notice] DeleteWorkspace successful, Duration 0.00 seconds\n.CreateWorkspace-[Notice] CreateWorkspace started\nCreateWorkspace-[Notice] CreateWorkspace successful, Duration 0.01 seconds\nLoadInstrument-[Notice] LoadInstrument started\nLoadInstrument-[Notice] LoadInstrument successful, Duration 0.00 seconds\nSetSample-[Notice] SetSample started\nSetSample-[Notice] SetSample successful, Duration 0.00 seconds\nDeleteWorkspace-[Notice] DeleteWorkspace started\nDeleteWorkspace-[Notice] DeleteWorkspace successful, Duration 0.00 seconds\n..CreateWorkspace-[Notice] CreateWorkspace started\nCreateWorkspace-[Notice] CreateWorkspace successful, Duration 0.00 seconds\nSetSample-[Notice] SetSample started\nSetSample-[Notice] SetSample successful, Duration 1.78 seconds\nInstrument-[Warning] In Instrument::getSource(). No source has been set.\nInstrument-[Warning] In Instrument::getSamplePos(). No SamplePos has been set.\nDeleteWorkspace-[Notice] DeleteWorkspace started\nDeleteWorkspace-[Notice] DeleteWorkspace successful, Duration 0.00 seconds\nMatplotlib is currently using agg, which is a non-GUI backend, so cannot show the figure.\n.CreateSampleWorkspace-[Notice] CreateSampleWorkspace started\nCreateSampleWorkspace-[Notice] CreateSampleWorkspace successful, Duration 0.00 seconds\nDeleteWorkspace-[Notice] DeleteWorkspace started\nDeleteWorkspace-[Notice] DeleteWorkspace successful, Duration 0.00 seconds\nMatplotlib is currently using agg, which is a non-GUI backend, so cannot show the figure.\n.CreateWorkspace-[Notice] CreateWorkspace started\nCreateWorkspace-[Notice] CreateWorkspace successful, Duration 0.00 seconds\nLoadInstrument-[Notice] LoadInstrument started\nLoadInstrument-[Notice] LoadInstrument successful, Duration 0.00 seconds\nSetSample-[Notice] SetSample started\nSetSample-[Notice] SetSample successful, Duration 0.00 seconds\nDeleteWorkspace-[Notice] DeleteWorkspace started\nDeleteWorkspace-[Notice] DeleteWorkspace successful, Duration 0.00 seconds\nMatplotlib is currently using agg, which is a non-GUI backend, so cannot show the figure.\n.CreateSampleWorkspace-[Notice] CreateSampleWorkspace started\nCreateSampleWorkspace-[Notice] CreateSampleWorkspace successful, Duration 0.01 seconds\nLoadSampleShape-[Notice] LoadSampleShape started\nLoadSampleShape-[Notice] LoadSampleShape successful, Duration 0.02 seconds\nDeleteWorkspace-[Notice] DeleteWorkspace started\nDeleteWorkspace-[Notice] DeleteWorkspace successful, Duration 0.00 seconds\nMatplotlib is currently using agg, which is a non-GUI backend, so cannot show the figure.\n..CreateWorkspace-[Notice] CreateWorkspace started\nCreateWorkspace-[Notice] CreateWorkspace successful, Duration 0.00 seconds\nDeleteWorkspace-[Notice] DeleteWorkspace started\nDeleteWorkspace-[Notice] DeleteWorkspace successful, Duration 0.00 seconds\nECreateWorkspace-[Notice] CreateWorkspace started\nCreateWorkspace-[Notice] CreateWorkspace successful, Duration 0.00 seconds\nLoadInstrument-[Notice] LoadInstrument started\nLoadInstrument-[Notice] LoadInstrument successful, Duration 0.00 seconds\nSetSample-[Notice] SetSample started\nSetSample-[Notice] SetSample successful, Duration 0.00 seconds\nDeleteWorkspace-[Notice] DeleteWorkspace started\nDeleteWorkspace-[Notice] DeleteWorkspace successful, Duration 0.00 seconds\nMatplotlib is currently using agg, which is a non-GUI backend, so cannot show the figure.\n.CreateWorkspace-[Notice] CreateWorkspace started\nCreateWorkspace-[Notice] CreateWorkspace successful, Duration 0.07 seconds\nSetSample-[Notice] SetSample started\nSetSample-[Notice] SetSample successful, Duration 0.00 seconds\nInstrument-[Warning] In Instrument::getSource(). No source has been set.\nInstrument-[Warning] In Instrument::getSamplePos(). No SamplePos has been set.\nDeleteWorkspace-[Notice] DeleteWorkspace started\nDeleteWorkspace-[Notice] DeleteWorkspace successful, Duration 0.00 seconds\nMatplotlib is currently using agg, which is a non-GUI backend, so cannot show the figure.\n.CreateWorkspace-[Notice] CreateWorkspace started\nCreateWorkspace-[Notice] CreateWorkspace successful, Duration 0.04 seconds\nSetSample-[Notice] SetSample started\nSetSample-[Notice] SetSample successful, Duration 0.00 seconds\nInstrument-[Warning] In Instrument::getSource(). No source has been set.\nInstrument-[Warning] In Instrument::getSamplePos(). No SamplePos has been set.\nDeleteWorkspace-[Notice] DeleteWorkspace started\nDeleteWorkspace-[Notice] DeleteWorkspace successful, Duration 0.00 seconds\nMatplotlib is currently using agg, which is a non-GUI backend, so cannot show the figure.\n.CreateWorkspace-[Notice] CreateWorkspace started\nCreateWorkspace-[Notice] CreateWorkspace successful, Duration 0.02 seconds\nSetSample-[Notice] SetSample started\nSetSample-[Notice] SetSample successful, Duration 5.60 seconds\nInstrument-[Warning] In Instrument::getSource(). No source has been set.\nInstrument-[Warning] In Instrument::getSamplePos(). No SamplePos has been set.\nDeleteWorkspace-[Notice] DeleteWorkspace started\nDeleteWorkspace-[Notice] DeleteWorkspace successful, Duration 0.00 seconds\nMatplotlib is currently using agg, which is a non-GUI backend, so cannot show the figure.\n.CreateSampleWorkspace-[Notice] CreateSampleWorkspace started\nCreateSampleWorkspace-[Notice] CreateSampleWorkspace successful, Duration 0.01 seconds\nLoadSampleShape-[Notice] LoadSampleShape started\nLoadSampleShape-[Notice] LoadSampleShape successful, Duration 0.02 seconds\nDeleteWorkspace-[Notice] DeleteWorkspace started\nDeleteWorkspace-[Notice] DeleteWorkspace successful, Duration 0.00 seconds\n.\n======================================================================\nERROR: test_sample_and_container_invalid (test_sample_shape.PlotSampleShapeTest)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqt/mantidqt/plotting/test/test_sample_shape.py&quot;, line 285, in test_sample_and_container_invalid\n    figure = sample_shape.plot_sample_container_and_components(workspace.name())\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqt/mantidqt/plotting/sample_shape.py&quot;, line 146, in plot_sample_container_and_components\n    return new_plot.create_plot(workspace_name)\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqt/mantidqt/plotting/sample_shape.py&quot;, line 53, in create_plot\n    raise Exception(&quot;Workspace must have attached Sample Shape or Environment&quot;)\nException: Workspace must have attached Sample Shape or Environment\n\n----------------------------------------------------------------------\nRan 35 tests in 23.171s\n\nFAILED (errors=1)\n\n          Start 1453: python.plots.UtilityTest.UtilityTest\n"
            }
        },
        {
            "test_name": "python.MuonQt5.model_fitting_view_test.model_fitting_view_test",
            "latest_failed_build": "1775",
            "agent_name": "Linux",
            "last_failed_outcome": "Flake",
            "pipeline_name": "pull_requests-conda-linux",
            "problem_count": 1,
            "past_outcome": {
                "build_keys": [
                    "1781",
                    "1780",
                    "1779",
                    "1778",
                    "1777",
                    "1776",
                    "1775",
                    "1774",
                    "1773",
                    "1772",
                    "1771",
                    "1770",
                    "1769",
                    "1768",
                    "1767",
                    "1766",
                    "1765",
                    "1764",
                    "1763",
                    "1762",
                    "1761",
                    "1760",
                    "1759",
                    "1758",
                    "1757",
                    "1756",
                    "1755",
                    "1754",
                    "1753",
                    "1752"
                ],
                "past_outcome_list": [
                    "None",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Flake",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "None",
                    "Passed",
                    "None",
                    "Passed",
                    "None",
                    "None",
                    "None",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "None",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "None"
                ]
            },
            "last_stack_traces": {
                "test_name": "python.MuonQt5.model_fitting_view_test.model_fitting_view_test",
                "build": "1775",
                "agent_name": "Linux",
                "trial": 1,
                "outcome": "Failed",
                "test_time": "5.58",
                "stacktrace": "FrameworkManager-[Notice] Welcome to Mantid 6.5.20221102.921\nFrameworkManager-[Notice] Please cite: http://dx.doi.org/10.1016/j.nima.2014.07.029 and this release: http://dx.doi.org/10.5286/Software/Mantid\nQStandardPaths: XDG_RUNTIME_DIR not set, defaulting to &#x27;/tmp/runtime-abc&#x27;\nlibEGL warning: DRI2: failed to open swrast (search paths /usr/lib64/dri)\nlibEGL warning: DRI2: failed to open swrast (search paths /usr/lib64/dri)\nlibEGL warning: DRI2: failed to open swrast (search paths /usr/lib64/dri)\nlibEGL warning: DRI2: failed to open swrast (search paths /usr/lib64/dri)\nQXcbIntegration: Cannot create platform OpenGL context, neither GLX nor EGL are enabled\n/jenkins_workdir/workspace/pull_requests-conda-linux/build/bin/site.py:26: DeprecationWarning: the imp module is deprecated in favour of importlib; see the module&#x27;s documentation for alternative uses\n  import imp  # Avoid import loop in Python 3\n/jenkins_workdir/workspace/pull_requests-conda-linux/mambaforge/envs/mantid-developer/lib/python3.8/site-packages/h5py/__init__.py:46: DeprecationWarning: `np.typeDict` is a deprecated alias for `np.sctypeDict`.\n  from ._conv import register_converters as _register_converters\nEEEEEEEE\n======================================================================\nERROR: test_that_add_results_table_name_will_add_a_result_table_to_the_workspace_selector (model_fitting_view_test.ModelFittingViewTest)\n----------------------------------------------------------------------\nAttributeError: &#x27;ModelFittingView&#x27; object has no attribute &#x27;y&#x27;\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/test/Muon/fitting_widgets/model_fitting/model_fitting_view_test.py&quot;, line 19, in setUp\n    self.view = ModelFittingView()\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/fitting_widgets/model_fitting/model_fitting_view.py&quot;, line 21, in __init__\n    super(ModelFittingView, self).__init__(parent)\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/fitting_widgets/basic_fitting/basic_fitting_view.py&quot;, line 29, in __init__\n    self.setupUi(self)\n  File &quot;&lt;string&gt;&quot;, line 31, in setupUi\nSystemError: &lt;built-in function connectSlotsByName&gt; returned a result with an error set\n\n======================================================================\nERROR: test_that_current_result_table_index_returns_the_expected_index (model_fitting_view_test.ModelFittingViewTest)\n----------------------------------------------------------------------\nAttributeError: &#x27;ModelFittingView&#x27; object has no attribute &#x27;y&#x27;\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/test/Muon/fitting_widgets/model_fitting/model_fitting_view_test.py&quot;, line 19, in setUp\n    self.view = ModelFittingView()\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/fitting_widgets/model_fitting/model_fitting_view.py&quot;, line 21, in __init__\n    super(ModelFittingView, self).__init__(parent)\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/fitting_widgets/basic_fitting/basic_fitting_view.py&quot;, line 29, in __init__\n    self.setupUi(self)\n  File &quot;&lt;string&gt;&quot;, line 31, in setupUi\nSystemError: &lt;built-in function connectSlotsByName&gt; returned a result with an error set\n\n======================================================================\nERROR: test_that_the_dataset_workspace_selector_is_hidden_by_default (model_fitting_view_test.ModelFittingViewTest)\n----------------------------------------------------------------------\nAttributeError: &#x27;ModelFittingView&#x27; object has no attribute &#x27;y&#x27;\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/test/Muon/fitting_widgets/model_fitting/model_fitting_view_test.py&quot;, line 19, in setUp\n    self.view = ModelFittingView()\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/fitting_widgets/model_fitting/model_fitting_view.py&quot;, line 21, in __init__\n    super(ModelFittingView, self).__init__(parent)\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/fitting_widgets/basic_fitting/basic_fitting_view.py&quot;, line 29, in __init__\n    self.setupUi(self)\n  File &quot;&lt;string&gt;&quot;, line 31, in setupUi\nSystemError: &lt;built-in function connectSlotsByName&gt; returned a result with an error set\n\n======================================================================\nERROR: test_that_the_view_is_enabled_if_results_tables_exist (model_fitting_view_test.ModelFittingViewTest)\n----------------------------------------------------------------------\nAttributeError: &#x27;ModelFittingView&#x27; object has no attribute &#x27;y&#x27;\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/test/Muon/fitting_widgets/model_fitting/model_fitting_view_test.py&quot;, line 19, in setUp\n    self.view = ModelFittingView()\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/fitting_widgets/model_fitting/model_fitting_view.py&quot;, line 21, in __init__\n    super(ModelFittingView, self).__init__(parent)\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/fitting_widgets/basic_fitting/basic_fitting_view.py&quot;, line 29, in __init__\n    self.setupUi(self)\n  File &quot;&lt;string&gt;&quot;, line 31, in setupUi\nSystemError: &lt;built-in function connectSlotsByName&gt; returned a result with an error set\n\n======================================================================\nERROR: test_that_the_view_is_not_enabled_if_results_tables_do_not_exist (model_fitting_view_test.ModelFittingViewTest)\n----------------------------------------------------------------------\nAttributeError: &#x27;ModelFittingView&#x27; object has no attribute &#x27;y&#x27;\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/test/Muon/fitting_widgets/model_fitting/model_fitting_view_test.py&quot;, line 19, in setUp\n    self.view = ModelFittingView()\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/fitting_widgets/model_fitting/model_fitting_view.py&quot;, line 21, in __init__\n    super(ModelFittingView, self).__init__(parent)\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/fitting_widgets/basic_fitting/basic_fitting_view.py&quot;, line 29, in __init__\n    self.setupUi(self)\n  File &quot;&lt;string&gt;&quot;, line 31, in setupUi\nSystemError: &lt;built-in function connectSlotsByName&gt; returned a result with an error set\n\n======================================================================\nERROR: test_that_update_result_table_names_will_select_the_previously_selected_item_if_it_still_exists (model_fitting_view_test.ModelFittingViewTest)\n----------------------------------------------------------------------\nAttributeError: &#x27;ModelFittingView&#x27; object has no attribute &#x27;y&#x27;\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/test/Muon/fitting_widgets/model_fitting/model_fitting_view_test.py&quot;, line 19, in setUp\n    self.view = ModelFittingView()\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/fitting_widgets/model_fitting/model_fitting_view.py&quot;, line 21, in __init__\n    super(ModelFittingView, self).__init__(parent)\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/fitting_widgets/basic_fitting/basic_fitting_view.py&quot;, line 29, in __init__\n    self.setupUi(self)\n  File &quot;&lt;string&gt;&quot;, line 31, in setupUi\nSystemError: &lt;built-in function connectSlotsByName&gt; returned a result with an error set\n\n======================================================================\nERROR: test_that_update_result_table_names_will_set_the_names_in_the_result_table_names_combobox (model_fitting_view_test.ModelFittingViewTest)\n----------------------------------------------------------------------\nAttributeError: &#x27;ModelFittingView&#x27; object has no attribute &#x27;y&#x27;\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/test/Muon/fitting_widgets/model_fitting/model_fitting_view_test.py&quot;, line 19, in setUp\n    self.view = ModelFittingView()\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/fitting_widgets/model_fitting/model_fitting_view.py&quot;, line 21, in __init__\n    super(ModelFittingView, self).__init__(parent)\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/fitting_widgets/basic_fitting/basic_fitting_view.py&quot;, line 29, in __init__\n    self.setupUi(self)\n  File &quot;&lt;string&gt;&quot;, line 31, in setupUi\nSystemError: &lt;built-in function connectSlotsByName&gt; returned a result with an error set\n\n======================================================================\nERROR: test_that_update_x_and_y_parameters_will_update_the_x_and_y_parameters (model_fitting_view_test.ModelFittingViewTest)\n----------------------------------------------------------------------\nAttributeError: &#x27;ModelFittingView&#x27; object has no attribute &#x27;y&#x27;\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/test/Muon/fitting_widgets/model_fitting/model_fitting_view_test.py&quot;, line 19, in setUp\n    self.view = ModelFittingView()\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/fitting_widgets/model_fitting/model_fitting_view.py&quot;, line 21, in __init__\n    super(ModelFittingView, self).__init__(parent)\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/fitting_widgets/basic_fitting/basic_fitting_view.py&quot;, line 29, in __init__\n    self.setupUi(self)\n  File &quot;&lt;string&gt;&quot;, line 31, in setupUi\nSystemError: &lt;built-in function connectSlotsByName&gt; returned a result with an error set\n\n----------------------------------------------------------------------\nRan 8 tests in 0.036s\n\nFAILED (errors=8)\n\n          Start 2188: python.MuonQt5.model_fitting_view_test.model_fitting_view_test\n"
            }
        },
        {
            "test_name": "AlgorithmsTest_FitPeaksTest",
            "latest_failed_build": "1761",
            "agent_name": "Linux",
            "last_failed_outcome": "Flake",
            "pipeline_name": "pull_requests-conda-linux",
            "problem_count": 1,
            "past_outcome": {
                "build_keys": [
                    "1781",
                    "1780",
                    "1779",
                    "1778",
                    "1777",
                    "1776",
                    "1775",
                    "1774",
                    "1773",
                    "1772",
                    "1771",
                    "1770",
                    "1769",
                    "1768",
                    "1767",
                    "1766",
                    "1765",
                    "1764",
                    "1763",
                    "1762",
                    "1761",
                    "1760",
                    "1759",
                    "1758",
                    "1757",
                    "1756",
                    "1755",
                    "1754",
                    "1753",
                    "1752"
                ],
                "past_outcome_list": [
                    "None",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "None",
                    "Passed",
                    "None",
                    "Passed",
                    "None",
                    "None",
                    "None",
                    "Passed",
                    "Passed",
                    "Flake",
                    "Passed",
                    "Passed",
                    "None",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "None"
                ]
            },
            "last_stack_traces": {
                "test_name": "AlgorithmsTest_FitPeaksTest",
                "build": "1761",
                "agent_name": "Linux",
                "trial": 1,
                "outcome": "Failed",
                "test_time": "0.42",
                "stacktrace": "FrameworkManager-[Notice] Welcome to Mantid 6.5.20221101.751\nFrameworkManager-[Notice] Please cite: http://dx.doi.org/10.1016/j.nima.2014.07.029 and this release: http://dx.doi.org/10.5286/Software/Mantid\nRunning 10 tests.Instrument-[Warning] In Instrument::getSource(). No source has been set.\nInstrument-[Warning] In Instrument::getSamplePos(). No SamplePos has been set.\nFitPeaks-[Notice] FitPeaks started\npure virtual method called\n\n********* UNHANDLED EXCEPTION *********\nSegmentation fault\n\n          Start  303: AlgorithmsTest_FitPeaksTest\n"
            }
        },
        {
            "test_name": "mantidqt_qt5.test_sliceviewer_imageinfowidget.test_sliceviewer_imageinfowidget",
            "latest_failed_build": "1760",
            "agent_name": "Linux",
            "last_failed_outcome": "Failed",
            "pipeline_name": "pull_requests-conda-linux",
            "problem_count": 1,
            "past_outcome": {
                "build_keys": [
                    "1781",
                    "1780",
                    "1779",
                    "1778",
                    "1777",
                    "1776",
                    "1775",
                    "1774",
                    "1773",
                    "1772",
                    "1771",
                    "1770",
                    "1769",
                    "1768",
                    "1767",
                    "1766",
                    "1765",
                    "1764",
                    "1763",
                    "1762",
                    "1761",
                    "1760",
                    "1759",
                    "1758",
                    "1757",
                    "1756",
                    "1755",
                    "1754",
                    "1753",
                    "1752"
                ],
                "past_outcome_list": [
                    "None",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "None",
                    "Passed",
                    "None",
                    "Passed",
                    "None",
                    "None",
                    "None",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Failed",
                    "Passed",
                    "None",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "None"
                ]
            },
            "last_stack_traces": {
                "test_name": "mantidqt_qt5.test_sliceviewer_imageinfowidget.test_sliceviewer_imageinfowidget",
                "build": "1760",
                "agent_name": "Linux",
                "trial": 3,
                "outcome": "Failed",
                "test_time": "1.88",
                "stacktrace": "/jenkins_workdir/workspace/pull_requests-conda-linux/build/bin/site.py:26: DeprecationWarning: the imp module is deprecated in favour of importlib; see the module&#x27;s documentation for alternative uses\n  import imp  # Avoid import loop in Python 3\n/jenkins_workdir/workspace/pull_requests-conda-linux/mambaforge/envs/mantid-developer/lib/python3.8/_collections_abc.py:720: MatplotlibDeprecationWarning: The global colormaps dictionary is no longer considered public API.\n  yield from self._mapping\nEEEEE\n======================================================================\nERROR: test_cursorAt_arguments_correct_when_not_mesh_and_not_transposed (test_sliceviewer_imageinfowidget.ImageInfoTrackerTest)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/mambaforge/envs/mantid-developer/lib/python3.8/unittest/mock.py&quot;, line 1325, in patched\n    return func(*newargs, **newkeywargs)\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqt/mantidqt/widgets/sliceviewer/test/test_sliceviewer_imageinfowidget.py&quot;, line 68, in test_cursorAt_arguments_correct_when_not_mesh_and_not_transposed\n    tracker = ImageInfoTracker(image=self.image,\nTypeError: __init__() missing 1 required positional argument: &#x27;presenter&#x27;\n\n======================================================================\nERROR: test_cursorAt_arguments_correct_when_not_mesh_and_transposed (test_sliceviewer_imageinfowidget.ImageInfoTrackerTest)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/mambaforge/envs/mantid-developer/lib/python3.8/unittest/mock.py&quot;, line 1325, in patched\n    return func(*newargs, **newkeywargs)\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqt/mantidqt/widgets/sliceviewer/test/test_sliceviewer_imageinfowidget.py&quot;, line 84, in test_cursorAt_arguments_correct_when_not_mesh_and_transposed\n    tracker = ImageInfoTracker(image=self.image,\nTypeError: __init__() missing 1 required positional argument: &#x27;presenter&#x27;\n\n======================================================================\nERROR: test_nonortho_transform_applied_when_mesh_and_transfrom_true (test_sliceviewer_imageinfowidget.ImageInfoTrackerTest)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqt/mantidqt/widgets/sliceviewer/test/test_sliceviewer_imageinfowidget.py&quot;, line 38, in test_nonortho_transform_applied_when_mesh_and_transfrom_true\n    tracker = ImageInfoTracker(image=self.image,\nTypeError: __init__() missing 1 required positional argument: &#x27;presenter&#x27;\n\n======================================================================\nERROR: test_nonortho_transform_not_applied_when_mesh_and_transfrom_false (test_sliceviewer_imageinfowidget.ImageInfoTrackerTest)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqt/mantidqt/widgets/sliceviewer/test/test_sliceviewer_imageinfowidget.py&quot;, line 27, in test_nonortho_transform_not_applied_when_mesh_and_transfrom_false\n    tracker = ImageInfoTracker(image=self.image,\nTypeError: __init__() missing 1 required positional argument: &#x27;presenter&#x27;\n\n======================================================================\nERROR: test_nonortho_transform_not_applied_when_not_mesh (test_sliceviewer_imageinfowidget.ImageInfoTrackerTest)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/mambaforge/envs/mantid-developer/lib/python3.8/unittest/mock.py&quot;, line 1325, in patched\n    return func(*newargs, **newkeywargs)\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqt/mantidqt/widgets/sliceviewer/test/test_sliceviewer_imageinfowidget.py&quot;, line 51, in test_nonortho_transform_not_applied_when_not_mesh\n    tracker = ImageInfoTracker(image=self.image,\nTypeError: __init__() missing 1 required positional argument: &#x27;presenter&#x27;\n\n----------------------------------------------------------------------\nRan 5 tests in 0.026s\n\nFAILED (errors=5)\n\n          Start 1719: python.functions.MsdPetersTest.MsdPetersTest\n"
            }
        }
    ]
}